core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -153,6 +198,60 @@ private[spark] class DiskBlockManager(conf: SparkConf, deleteFilesOnStop: Boolea
     }
   }
 
+  /**
+   * Get the list of configured local dirs storing merged shuffle blocks created by executors
+   * if push based shuffle is enabled. Note that the files in this directory will be created
+   * by the external shuffle services. We only create the merge_manager directories and
+   * subdirectories here because currently the shuffle service doesn't have permission to
+   * create directories under application local directories.
+   */
+  private def createLocalDirsForMergedShuffleBlocks(conf: SparkConf): Array[File] = {
```

Comment:
This change is for the code refactor as Mridul commented earlier.

Reply:
@zhouyejoe The earlier comment must be because the PR didn't have latest code and it was needed for initializing `activeMergedShuffleDirs`. There is no need for `activeMergedShuffleDirs`. As mentioned in the other comment it is not being used anywhere in `DiskBlockManager`. The dirs are being passed by the methods so why does this need to return the files.
cc @mridulm